src/courseware/course/sequence/sequence-navigation/SequenceNavigationTabs.test.jsx

```diff
@@ -60,10 +63,35 @@ describe('Sequence Navigation Tabs', () => {
       await fireEvent.click(dropdownToggle);
     });
     const dropdownMenu = container.querySelector('.dropdown');
-    const dropdownButtons = getAllByRole(dropdownMenu, 'tab');
+    const dropdownButtons = getAllByRole(dropdownMenu, 'tabpanel');
+
     expect(dropdownButtons).toHaveLength(unitBlocks.length);
     expect(getAllByRole(dropdownMenu, 'button')).toHaveLength(1);
     expect(screen.getByRole('button', { name: `${activeBlockNumber} of ${unitBlocks.length}` }))
       .toHaveClass('dropdown-toggle');
   });
+
+  it('focuses buttons after pressing ArrowRight or ArrowLeft keys', async () => {
+    useIndexOfLastVisibleChild.mockReturnValue([0, null, null]);
+    render(<SequenceNavigationTabs {...mockData} />);
+
+    const firstUnitButton = screen.getAllByRole('tabpanel')[0];
+    firstUnitButton.focus();
+
+    await act(async () => {
+      await userEvent.keyboard('{ArrowRight}');
```

Comment:
[nit]: It seems to me that the `userEvent` in its internal implementation is already wrapped in `act()`? Could you remove `act()` and rewrite the test without it?

Some info
- https://github.com/testing-library/user-event/issues/497
- https://github.com/testing-library/user-event/discussions/906

Reply:
Fixed